File: tests/testIn/test6.c
#include <stdio.h>
#include <time.h>

int main() {
    long i;
    long total = 0;
    clock_t start_time, end_time;

    start_time = clock();

    for (i = 1; i < 10000000; i++) {
        total += i;
    }

    end_time = clock();

    printf("C: The sum is %ld\n", total);
    printf("C: Time taken = %lf seconds\n", (double)(end_time - start_time) / CLOCKS_PER_SEC);

    return 0;
}

File: tests/testIn/test6.py
import time

def sum_integers():
    total = 0
    for i in range(1, 10**7):
        total += i
    return total

start_time = time.time()
result = sum_integers()
end_time = time.time()

print(f"Python: The sum is {result}")
print(f"Python: Time taken = {end_time - start_time} seconds")
